maintainers/maintainer-list.nix

```diff
@@ -3065,6 +3065,12 @@
     githubId = 8182846;
     name = "Francesco Gazzetta";
   };
+  figsoda = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add figsoda``.